airbyte-cdk/bulk/toolkits/load-object-storage/src/main/kotlin/io/airbyte/cdk/load/pipline/object_storage/ObjectLoaderPipeline.kt

```diff
@@ -101,11 +101,11 @@ class ObjectLoaderPipeline<K : WithStream, T : RemoteObject<*>>(
             return if (hasFileTransfer(catalog)) {
```

Comment:
Can we combine this condition with what you're adding? Otherwise looks like we have 3 ways to check if file transfer is enabled

Reply:
Done.